lib/src/code_generator/func_type.dart

```diff
@@ -54,6 +55,21 @@ class FunctionType extends Type {
       p.type.addDependencies(dependencies);
     }
   }
+
+  void addParameterNames(List<String> names) {
+    if (names.length != parameters.length) {
+      return;
+    }
+    final paramNamer = UniqueNamer({});
```

Comment:
What's the correct way to rename parameters here so that none will clash with a dart keyword?

Reply:
UniqueNamer will handle all dart keywords defined in the keywords.dart file